geniusyield-market-maker/src/GeniusYield/MarketMaker/MakerBot.hs

```diff
@@ -111,15 +111,12 @@ buildAndSubmitActions user@User {uColl, uStakeCred} providers netId ua di = flip
     let handlerCommon ∷ Exception e => e → IO ()
         handlerCommon = logWarn . displayException
 
-        be ∷ BuildTxException → IO ()
-        be = handlerCommon
-
         se ∷ SubmitTxException → IO ()
         se = handlerCommon
 
         me ∷ GYTxMonadException → IO ()
         me = handlerCommon
-     in [Handler be, Handler se, Handler me]
+     in [Handler se, Handler me]
```

Comment:
`BuildTxException` (now renamed to GYBuildTxError`) is part of `GYTxMonadException`.